pages/DSNP/Overview.md

```diff
@@ -1,10 +1,32 @@
 # DSNP Specification
 __Version pre-1.2.0__
 
-DSNP is a social networking protocol designed to run on a blockchain.
+DSNP (Decentralized Social Networking Protocol) is a social networking protocol designed to run on a blockchain.
 It specifies a set of social primitives along with requirements for interoperability.
 Go to [Implementations](../Implementations.md) for specifics on how DSNP is implemented on a specific blockchain.
 
+## Overview
+
+### What is a DSNP System?
+
+A DSNP system is a state machine which generates an ongoing, publicly observable and verifiable stream of state change records in response to public input.
+This specification defines a set of operations (the DSNP [Operations](Operations.md)) that a compliant DSNP implementation MUST make available.
+For each DSNP Operation, the DSNP specification defines the data to be provided as input, the data that will be generated as output, and the state change records (the DSNP [State Change Records](Records.md)) that will be observable as a result of the Operation, depending on the input and prior state of the system.
+
+The nature of blockchain technology means that all DSNP Operations are potentially asynchronous; that is, DSNP State Change Records need not be created immediately upon invoking a DSNP Operation, and may appear in the stream of state changes at any subsequent time.
+A DSNP Operation can therefore only be confirmed to have completed successfully by observing the stream of newly-generated DSNP Records.
+ 
+All state changes MUST be visible to all participants in a DSNP system.
+In practical terms, this means that all users of a given DSNP system exist within a single overarching social graph, and must be able to see and participate in all public discourse on the network.
```

Comment:
So we are explicitly saying that you cannot be a partial participant in a DSNP system and be compliant, i.e. you can't just post GraphChange messages and keep the rest "to yourself," away from the implementation?